src/docfx/lib/git/GitCommitProvider.cs

```diff
@@ -9,12 +9,14 @@ namespace Microsoft.Docs.Build
 {
     internal sealed class GitCommitProvider : IDisposable
     {
-        private readonly ConcurrentDictionary<string, FileCommitProvider> _fileCommitProvidersByRepoPath = new ConcurrentDictionary<string, FileCommitProvider>();
+        private readonly ConcurrentDictionary<string, FileCommitProvider>
```

Comment:
nit: vertically align the type signiture

```suggestion
        private readonly ConcurrentDictionary<string, FileCommitProvider> _fileCommitProvidersByRepoPath
                       = new  ConcurrentDictionary<string, FileCommitProvider>
```